de_workloads/data_processing/silver/silver.yml

```diff
@@ -14,64 +8,53 @@ pr:
     - 'main'
   paths:
     include:
-      - ingest/jobs/Ingest_AzureSql_Example/adfpipeline/*
-      - ingest/jobs/Ingest_AzureSql_Example/tests/*
-      - ingest/jobs/Ingest_AzureSql_Example/spark-jobs/*
-      - ingest/jobs/Ingest_AzureSql_Example/ingest-azuresql-example.yml
+      - de_workloads/data_processing/silver/*
 
 trigger:
   branches:
-   
     include:
       - 'main'
   paths:
     include:
-      - ingest/jobs/Ingest_AzureSql_Example/adfpipeline/*
-      - ingest/jobs/Ingest_AzureSql_Example/tests/*
-      - ingest/jobs/Ingest_AzureSql_Example/spark-jobs/*
-      - ingest/jobs/Ingest_AzureSql_Example/ingest-azuresql-example.yml
+      - de_workloads/data_processing/silver/*
 
 
 variables:
   - template: ../../../de_build/job-pipeline-vars.yml
   - template: ../../../de_build/air-data-testing-vars.yml
   - template: ../../../build/version-data-vars.yml
   - name: job  #update job name
-    value: 'Ingest_AzureSql_Example'
+    value: 'silver'
+  - name: jobtype  #update job name
+    value: 'data_processing'
+  - name: include_data_quality  # update data quality option
+    value: 'true'
+  - name: self_repo_job_dir
+    value: '$(self_repo_dir)/de_workloads/$(jobtype)/$(job)'
   - name: self_repo_adf_dir
-    value: '$(self_repo_dir)/ingest/jobs/$(job)/adfpipeline'
+    value: '$(self_repo_job_dir)/data_factory'
   - name: test_unit_path
-    value: '$(self_repo_dir)/ingest/jobs/$(job)/tests/unit/'
+    value: '$(self_repo_job_dir)/tests/unit/'
   - name: test_end_to_end_path
-    value: '$(self_repo_dir)/ingest/jobs/$(job)/tests/end_to_end/features/'
+    value: '$(self_repo_job_dir)/tests/end_to_end/features/'
   - name: junit_path
-    value: '$(self_repo_dir)/ingest/jobs/$(job)/junit/'
+    value: '$(self_repo_job_dir)/junit/'
   - name: self_repo_sparkjob_dir
-    value: '$(self_repo_dir)/ingest/jobs/$(job)/spark-jobs'
-  - name: overrideParameters
-    value: '-factoryName $(datafactoryname)
-     -adlsStorageAccountName $(blob_adls_storage)
-     -blobStorageAccountName $(blob_configStorage)
-     -databricksHost $(databricksHost)
-     -databricksWorkspaceResourceId $(databricksWorkspaceResourceId)
-     -ls_AzureSql_ExampleSource_connectionString $(sql_connection)
-     -ls_Blob_ConfigStore_properties_typeProperties_serviceEndpoint $(Blob_ConfigStore_serviceEndpoint)
-     -ls_ADLS_DataLake_properties_typeProperties_url $(ADLS_DataLake_URL)
-     -ls_KeyVault_properties_typeProperties_baseUrl $(KeyVault_baseURL)'
-  - name: self_repo_adf_src  #update job name
-    value: "$(jobtype)/jobs/$(job)/adfpipeline"
-  - name: jobtype  #update job name
-    value: 'ingest'
+    value: '$(self_repo_job_dir)/spark-jobs'
+  - name: self_repo_adf_src
+    value: "de_workloads/$(jobtype)/$(job)/data_factory"
+  - name: self_repo_blob_config
+    value: '$(self_repo_job_dir)/config'
```

Comment:
Do we want to always upload all configs for silver? Right now this is only Data Quality config, and DQ is optional.

Reply:
I think that the best way would be to keep the pipelines separate: DQ in a separate directory from non-DQ, e.g.
- Ingest_AzureSql_Example
- Ingest_AzureSql_ExampleDQ
etc. But feel free to resolve it for now, we can change it later if we agree on that. I think it also would be easier for preparing the templates.

Reply:
Sorry, I edited my previous comment :)